src/app/dim-ui/Sheet.tsx

```diff
@@ -348,3 +349,72 @@ function isInside(element: HTMLElement, className: string) {
   }
   return false;
 }
+
+// After this many pixels of dragging in either direction, we consider ourselves to be part of a scrolling gesture.
+const HORIZ_SCROLL_DRAG_THRESHOLD = 20;
+
+export function HorizontalScrollContainer({ children }: PropsWithChildren) {
```

Comment:
Maybe should rename this to 

```suggestion
export function SheetHorizontalScrollContainer({ children }: PropsWithChildren) {
```

but it could be used anywhere. It is just that the sheet needs it.

Reply:
Also, add documentation